analysis/test_cases.go

```diff
@@ -26,7 +26,7 @@ var Tier2TestCases = []TC{
 	SeamBooking,
 }
 
-// Tier 3 Analysis with credentials test cases - should work
+// Tier 3 tests - should work and should be executed on internal/private infrastructure
```

Comment:
Tier runs here refer to analysis test cases only. 
See https://github.com/konveyor/go-konveyor-tests?tab=readme-ov-file#tier-3 for the general TIER3 execution